support-frontend/assets/components/product/productOption.jsx

```diff
@@ -95,34 +182,48 @@ function ProductOption(props: Product) {
     }
   }, [hasBeenSeen]);
 
+  const productOptionMargin = props.label && css`
+  ${until.tablet} {
+    /* calculation belows are based on productOptionHighlight text size, line height and padding */
+    &:first-of-type {
+      margin-top: calc((20px * 1.5) + 8px) !important;
+    }
+    /* 16px alloted for margin between product options when a label is present */
+    &:not(first-of-type) {
+      margin-top: calc((20px * 1.5) + 8px + 16px) !important;
+    }
+    }
+  `;
+
   return (
-    <div ref={setElementToObserve} css={[productOption, props.cssOverrides]}>
-      <div>
+    <div ref={setElementToObserve} css={[productOption, props.cssOverrides, productOptionMargin]}>
+      <div css={productOptionVerticalLine}>
         <h3 css={[productOptionTitle, productOptionUnderline]}>{props.title}</h3>
         {props.label && <span css={productOptionHighlight}>{props.label}</span>}
         {props.children && props.children}
       </div>
-      <div>
+      <div css={productOptionVerticalLine}>
         <p css={[productOptionOfferCopy, productOptionUnderline]}>
           {props.offerCopy}
         </p>
       </div>
-      <div>
+      <div css={priceCopyGridPlacement}>
         {/* role="text" is non-standardised but works in Safari. Reads the whole section as one text element */}
         {/* eslint-disable-next-line jsx-a11y/aria-role */}
         <p role="text" css={productOptionPriceCopy}>
           <span css={productOptionPrice}>{props.price}</span>
           {props.priceCopy}
         </p>
       </div>
-      <div>
+      <div css={buttonDiv}>
         <ThemeProvider theme={buttonReaderRevenue}>
           <LinkButton
+            css={button}
             href={props.href}
             onClick={props.onClick}
-            aria-label={`${props.title}- ${props.buttonCopy}`}
+            aria-label={`${props.title}- ${props.buttonCopy} now`}
           >
-            {props.buttonCopy}
+            {props.buttonCopy}<span css={buttonCopySpan}>&nbsp;now</span>
```

Comment:
Is this `<span>` appending "now" to the button copy necessary? I think it'd be easier to find where the button copy string is defined, before it's passed down in the props, and to just add "now" to the button copy string. Plus, this is a shared Component, so hard coding "now" on the end here can have unintended side affects. EG. The Guardian Weekly CTA on the pricing cards becomes:

<img width="204" alt="Screenshot 2021-09-20 at 09 08 20" src="https://user-images.githubusercontent.com/1590704/133973068-ab5047b9-6ac4-4e4e-acd9-955d1af94df9.png">

Reply:
Ah yes I'd missed where it was shared. With the latest design the text for the button now changes for different screen sizes so had opted to conditionally include the "now" copy using media queries but will change this now.

Reply:
OK I see, good to know! I'd question whether the inclusion of "now" is really essential for the objective of this piece of work - to launch this Newspaper campaign, I'd say the current wording on the CTA without the "now" is fine, and if we do want to implement different wording at different breakpoints we create a separate card/PR to do this.